server/app.js

```diff
@@ -37,23 +42,28 @@ require("./config/passport")(passport);
 // Routes
 app.use("/", indexRouter);
 app.use("/ping", pingRouter);
+// Login Routes
 app.use("/auth/register", registerRouter);
 app.use("/auth/login", loginRouter);
+// Polls backend routes
+app.use("/polls/add", addPollRouter);
+app.use("/polls/delete", deletePollRouter);
+app.use("/polls/update", updatePollRouter);
 
 // catch 404 and forward to error handler
-app.use(function(req, res, next) {
-  next(createError(404));
+app.use(function (req, res, next) {
+	next(createError(404));
 });
 
 // error handler
-app.use(function(err, req, res, next) {
-  // set locals, only providing error in development
-  res.locals.message = err.message;
-  res.locals.error = req.app.get("env") === "development" ? err : {};
-
-  // render the error page
-  res.status(err.status || 500);
-  res.json({ error: err });
+app.use(function (err, req, res, next) {
+	// set locals, only providing error in development
+	res.locals.message = err.message;
+	res.locals.error = req.app.get("env") === "development" ? err : {};
+
+	// render the error page
+	res.status(err.status || 500);
+	res.json({ error: err });
```

Comment:
Looks like we changed the beautifier and linter being used. We'll need to be on the same page for these settings - I rather keep the smaller size tabs though than change it to this @tgnemecek @dotslash227 @Hakeemmidan

Reply:
@skassam21 Can you share your prettier config file? And we all can load it and have it consistent.